doc/admx/en-US/DesktopAppInstaller.adml

```diff
@@ -6,82 +6,82 @@
   <description>App Installer</description>
   <resources>
     <stringTable>
-      <string id="AppInstaller">Desktop App Installer</string>
-      <string id="EnableAppInstaller">Enable App Installer</string>
+      <string id="AppInstaller">Desktop Windows Package Manager</string>
+      <string id="EnableAppInstaller">Enable Windows Package Manager</string>
       <string id="EnableAppInstallerExplanation">This policy controls whether the Windows Package Manager can be used by users.
 
 If you enable or do not configure this setting, users will be able to use the Windows Package Manager.
 
 If you disable this setting, users will not be able to use the Windows Package Manager.</string>
-      <string id="EnableSettings">Enable App Installer Settings</string>
+      <string id="EnableSettings">Enable Windows Package Manager Settings</string>

```

Comment:
I am proposing we make a distinction between on which policies affect winget and which ones affect App Installer here, but we can drop that change. The important part in this PR is the new policies.